src/internal/plugins/controllers/drawers.ts

```diff
@@ -48,6 +51,23 @@ export class DrawersController {
     this.scheduleUpdate();
   };
 
+  updateDrawer = (config: UpdateDrawerConfig) => {
+    const updateDrawerConfig = config as Partial<DrawerConfig>;
+    if (updateDrawerConfig?.onResize || updateDrawerConfig?.mountContent || updateDrawerConfig?.unmountContent) {
```

Comment:
Same here. Let's validate that the `config` argument only contains known dynamic properties